.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Self checklist**
+
+ - [ ] This issue hasn't been already created
+ - [ ] Everything is sound and clear, all sections are filled-up
+ - [ ] Relevant labels have been added
+
+# Expected Behavior (short explanation)
+
+Please describe the behavior you are expecting
+
+# Current Behavior
+
+What is the current behavior?
+
+# Failure Information
+
+Please help provide information about the failure.
+
+## Steps to Reproduce
+
+1. step 1
+2. step 2
+3. you get it...
+
+## Context
+
+Please provide any relevant information about your setup. 
+
+* Inmanta version:
+* Operating System:
+* Python version:
+
+## Failure Logs
```

Comment:
how is failure logs different from Failure Information and context

Reply:
*Failure logs* should contain any relevant trace (in case of error), debug execution steps (if relevant) or any output from the software (here `inmanta`).

The *context* might contain global information (like if the error occured since the last system update).

Both *failure logs* and *context* are a part of *failure information*

Reply:
Is it more clear now? https://github.com/inmanta/inmanta/pull/2416/commits/5090b071922f322cee876f9d6288b7279514d3b2

Reply:
Why is this not part of the Current Behavior or Steps to Reproduce (both seem natural places to put a stacktrace?)

Reply:
Structure has been updated here: https://github.com/inmanta/inmanta/pull/2416/commits/109c0803e5788a67f3414672083d954ea8283964